docs/application/dotnet/guides/nui/widget.md

```diff
@@ -0,0 +1,257 @@
+# NUI Widget Application
+
+[NUIWidgetApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html)  is to show a small version of application on home screen.
+
+**Figure: NUI widget application**
+
+![NUI widget application](./media/dali_widget_application.png)
+
+<br>
+The main widget application features include:
+
+- Creating widget applications
+
+  You can [Create a NUI Widget Application](#create-a-nui-widget-application)
+, which usually has 1 process for maintaining the main loop. Within the process, the framework can be create widget instance that can share the same resources.
+  The Widget application can also share data with other applications.
+
+- Managing widget instances
+  
+  Whenever a widget instance is requested, the framework creates one.
+  You can manage the widget instances by updating or destroying them, or by retrieving information about them.
+
+- Managing the life-cycle
+  
+  You can manage the widget instance life-cycle through callback methods that are triggered as the instance state changes.
```

Comment:
Suggetsion: Will it be good to provide a link to the widget life-cycle ?https://docs.tizen.org/application/dotnet/guides/applications/widget-app#instance

Please ignore this comment if invalid.